pkg/juicefs/mount/pod_mount.go

```diff
@@ -400,7 +400,14 @@ func (p *PodMount) waitUtilJobCompleted(ctx context.Context, jobName string) err
 		}
 		if util.IsJobCompleted(job) {
 			klog.V(5).Infof("waitUtilJobCompleted: Job %s is completed", jobName)
-			return nil
+			if util.IsJobShouldBeRecycled(job) {
+				// try to delete job
+				klog.Infof("job %s completed but not be recycled automatically, delete it", jobName)
+				if err := p.K8sClient.DeleteJob(ctx, jobName, jfsConfig.Namespace); err != nil {
+					klog.Errorf("delete job %s error %v", jobName, err)
+					return err
```

Comment:
This `err` should always be returned even if it's nil, otherwise, there will be a dead loop.

Reply:
done